storage/tree/node_id2.go

```diff
@@ -24,8 +24,8 @@ import "fmt"
 // code has migrated.
 type NodeID2 struct {
 	path string
-	last byte
-	bits uint8
+	last byte  // Invariant: Lowest (8-bits) bits are unset.
```

Comment:
last is `byte` which is 8 bits, making this comment confusing.  is `last` always 0 ?

Reply:
`last` is not always 0.

Example: An 11-bit ID `1010,1010,101` is interpreted as follows:

- `path` contains 1 byte, which is `1010,1010`.
- `last` is `1010,0000`.
- `bits` is 3, so effectively the upper 3 bits of `last` are used (`101`).

Reply:
I might put this example straight into the comments if it looks good to you :)

Reply:
Done.